src/goto-symex/field_sensitivity.h

```diff
@@ -15,7 +15,36 @@ class namespacet;
 class goto_symex_statet;
 class symex_targett;
 
-/// \brief Control granularity of object accesses
+/// Control granularity of object accesses
+///
+/// [Field sensitivity](\ref field_sensitivityt) is a transformation of the
+/// instructions goto-program which mainly replaces some expressions by symbols
```

Comment:
The interface of `field_sensitivityt` doesn't mention instructions, it transforms expressions.

Reply:
Yes but field_assignments mentions assignments and assignments are not expressions. Should I say expressions and assignments?